doc/api/url.md

```diff
@@ -835,10 +833,9 @@ The legacy urlObject (`require('url').Url`) is created and returned by the
 #### urlObject.auth
 
 The `auth` property is the username and password portion of the URL, also
-referred to as "userinfo". This string subset follows the `protocol` and
-double slashes (if present) and precedes the `host` component, delimited by an
-ASCII "at sign" (`@`). The format of the string is `{username}[:{password}]`,
-with the `[:{password}]` portion being optional.
+referred to as _userinfo_. This string subset follows the `protocol` and
+double slashes (if present) and precedes the `host` component, delimited by `@`.
+The format of the string is `{username}[:{password}]`.
```

Comment:
I think it is actually important to keep it the way it was. Especially for people who are not that familiar yet with the optional notation. In this specific case I would say it is hard to see that that part is meant to be "optional".